tests/test_property.py

```diff
@@ -77,10 +101,71 @@ def test_to_graph_should_return_has_type() -> None:
     assert _isomorphic
 
 
-def test_to_graph_should_return_blank_node() -> None:
-    """It returns a property graph as blank node isomorphic to spec."""
+def test_to_graph_should_return_has_type_blank_node_property_identifier() -> None:
+    """It returns a has_type graph isomorphic to spec."""
+    property = Property()
+    property.identifier = "http://example.com/properties/1"
+
+    modelelement = ModelElement()
+    property.has_type.append(modelelement)
+
+    src = """
+        @prefix dct: <http://purl.org/dc/terms/> .
+        @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+        @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+        @prefix dcat: <http://www.w3.org/ns/dcat#> .
+        @prefix modelldcatno: <https://data.norge.no/vocabulary/modelldcatno#> .
+
+        <http://example.com/properties/1> a modelldcatno:Property ;
+        modelldcatno:hasType [ ] .
+
+        """
+    g1 = Graph().parse(data=property.to_rdf(), format="turtle")
+    g2 = Graph().parse(data=src, format="turtle")
+
+    _isomorphic = isomorphic(g1, g2)
+    if not _isomorphic:
+        _dump_diff(g1, g2)
+        pass
+    assert _isomorphic
+
+
+def test_to_graph_should_return_has_type_blank_node_modelelement_identifier() -> None:
+    """It returns a has_type graph isomorphic to spec."""
     property = Property()
 
+    modelelement = ModelElement()
+    modelelement.identifier = "http://example.com/modelelements/1"
+    property.has_type.append(modelelement)
+
+    src = """
+          @prefix dct: <http://purl.org/dc/terms/> .
+        @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+        @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+        @prefix dcat: <http://www.w3.org/ns/dcat#> .
+        @prefix modelldcatno: <https://data.norge.no/vocabulary/modelldcatno#> .
+
+        [] a modelldcatno:Property .
```

Comment:
Igjen; usikker på om det er lov å legge til blanke noder slik

Reply:
Og det er uansett feil, dette er bare to ressurser som ikke har noe med hverandre å gjøre...

Reply:
```
[ a    modelldcatno:Property; 
  modelldcatno:hasType    <http://example.com/modelelements/1>
] .
```